### DISCUSSION TITLE
How to save text as one txt file a post?

### DISCUSSION THREAD

**Discussion by startingstuffup** [TumblThreeApp/TumblThree#531] [Q&A (Question & Answer)]
I'm trying to backup a writing blog and a post has 1-2 pictures and long text. How do I save the text individually as a txt file instead of just putting it all in one big file? I already found out how to group pictures together with filenames, so that's not an issue, but the texts are a mess. And additionally, I'd like to put the same constraints on the txt file so it'll be grouped with the pictures from within that post (Like similarly named with the images it's associated with)

Edit: tumblr-utils seem to be capable for this, but it's not the best for hidden blogs

**Comment by startingstuffup** [comment]
So... I guess I'll assume this isn't an option right now?

**Comment by thomas694** [comment]
Currently, this is not possible.
